workshop1/04-testing-your-code.md

```diff
@@ -41,13 +41,21 @@ This set of instructions are using [jest-dom](https://github.com/testing-library
    import { PetCard } from './PetCard';
    ```
 
-5. Add an import statement to import the `Pet` data model we created in the previous step. Again make sure that the path matches the location where the model is located relative to the `Petcard.test.tsx` file.
+5. Btw, to be able to import PetCart, you will need to export it first. Add the export keyword to the PetCard class in the `PetCard.tsx` file to achieve this:
```

Comment:
Ah, we already have an `export default PetCard;` in `PetCard.tsx`, so we should probably fix this instead by changing the suggested import in the test file from

```tsx
import { PetCard } from './PetCard';
```

to

```tsx
import PetCard from './PetCard';
```

i.e. to import the default export rather than a named export from that module.

Reply:
Addressed in 365b234.